osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -56,6 +55,8 @@ public WorkingBeatmap Beatmap
         public BackgroundModeBeatmap(WorkingBeatmap beatmap)
         {
             Beatmap = beatmap;
+            if (beatmap == null)
+                Add(background = new Background(@"Backgrounds/bg1"));

```

Comment:
move this logic to line29 and combine with the rest of the logic that is adding the background (and apply the same blur logic etc. it should behave exactly the same.